src/crimson/net/Fwd.h

```diff
@@ -26,6 +26,14 @@ using auth_proto_t = int;
 class Message;
 using MessageRef = boost::intrusive_ptr<Message>;
 
+struct MMonMap;
+struct MAuthReply;
+struct MMonSubscribeAck;
+struct MMonGetVersionReply;
+struct MMonCommandAck;
+struct MLogAck;
+struct MConfig;
+
```

Comment:
could these live in MonClient.h or a separate mon/Fwd.h to keep mon details out of crimson/net?

Reply:
agreed. they don't belong to `net`. will move them into `MonClient.h` .

Reply:
@cbodley updated and repushed.